km_api/know_me/tests/models/test_km_user_accessor_model.py

```diff
@@ -0,0 +1,20 @@
+from know_me import models
+
+
+def test_create(km_user_factory, user_factory):
+    """
+    Test creating a KMUserAccessor.
+    """
+    models.KMUserAccessor.objects.create(
+        km_user=km_user_factory(),
+        user_with_access=user_factory())
```

Comment:
We prefer to provide values for all the fields so we ensure that we didn't miss any.